app/services/idv/steps/back_image_step.rb

```diff
@@ -36,6 +36,8 @@ def handle_document_verification_failure(get_results_response)
         extra = get_results_response.to_h.merge(
           notice: I18n.t('errors.doc_auth.general_info'),
         )
+        Funnel::DocAuth::LogDocumentError.call(user_id,
+                                               get_results_response&.result_code&.name.to_s)
```

Comment:
So `result_code` is an enum of 6 things and it's not that detailed.

Do we want to do something that processes the `"Alerts"` key instead? https://github.com/18F/identity-idp/blob/master/spec/fixtures/acuant_responses/get_results_response_failure.json#L1-L17

Reply:
The enum makes it perfect.  I wanted them to be categorized so I could quickly run analytics on how many users got unknown vs failed or attention etc.  If we want details for a particular user we can go to cloudwatch